clearwater-infrastructure/etc/bash.bashrc.clearwater

```diff
@@ -1,22 +1,30 @@
+# Read out the node types on the server from the installed packages
 type=$(. /etc/clearwater/config
        {
+         # List all installed packages for CentOS or Debian. rpm returns an unordered list, dpkg-query is already sorted
```

Comment:
You can put explanations of the commands themselves in comments within the if statement. What's interesting to know here is why you're running `if which yum`. So you might say:

`# Different operating systems use their own package management systems. We assume that if yum is installed we are running on CentOS and working with RPMs, and otherwise we are running on Ubuntu and working with .deb packages.`